### ISSUE TITLE
structure alphabets

### ISSUE THREAD

**Issue by h-2** [seqan/seqan3#73]
I think we should add the structure formats that are alphabets also to the alphabet module.

The following would make sense from my POV:
```
alphabet/structure/all.hpp
alphabet/structure/concept.hpp
alphabet/structure/dot_bracket3.hpp    // rna: . ( )
alphabet/structure/wuss18.hpp    // rna: .,;<>(){}[]AaBb.-_
alphabet/structure/dssp9.hpp   // protein: HGIEBTSCX
```
Wuss could probably be bigger as well since all character pairs are allowed, I think...

@joergi-w What do you think? Can you work on this?

**Comment by joergi-w** [comment]
The structure alphabets are introduced in pull request #92.

**Comment by h-2** [comment]
implemented via #92
